buildtools/pom.xml

```diff
@@ -40,6 +40,21 @@
       <artifactId>testng</artifactId>
       <version>6.13.1</version>
     </dependency>
+    <dependency>
```

Comment:
If every module depends on `buildtool`, everyone will depend on log4j

Reply:
the log4j are already default to all modules in test scope, buildtools is also default to all modules in test scope, since it is shipping a default log4j2.yaml for testing. so it is making sense to move them to buildtools.